### PR TITLE
allow loading of reflection background image

### PR DESCRIPTION
**Pull request by unacao**

### CODE REVIEW

File: modules/dreamview/frontend/src/renderer/ground.js [hunk 1]
```diff
@@ -23,20 +23,71 @@ export default class Ground {
...
-        const { xres, yres, mpp } = PARAMETERS.ground;
+        if (!this.render(coordinates)) {
+            return false;
+        }
+
+        this.initialized = true;
+        return true;
+    }
+
+    update(world, coordinates, scene) {
```
Comment (siyangy) [inline modules/dreamview/frontend/src/renderer/ground.js]: is this function still useful?

File: modules/dreamview/frontend/src/renderer/ground.js [hunk 2]
```diff
@@ -23,20 +23,71 @@ export default class Ground {
...
+    }
+
+    updateImage(coordinates, mapName) {
+        if (!this.mesh) {
+            return;
+        }
+        const dir = this.titleCaseToSnakeCase(mapName);
+        /*import(`assets/map_data/${dir}/background.jpg`)
```
Comment (siyangy) [inline modules/dreamview/frontend/src/renderer/ground.js]: is it okay to remove commented code?

File: modules/dreamview/frontend/src/renderer/ground.js [hunk 3]
```diff
@@ -23,20 +23,71 @@ export default class Ground {
...
+        .then(ground => {
+            console.log("updating ground image with " + dir);
+            loadTexture(ground, texture => {
+                this.mesh.material.map = texture;
+                this.render(coordinates, dir);
+            });
+        })*/
+
+        const server = 'http://' + window.location.hostname + ':8888';
+        const imgUrl = server + '/assets/map_data/' + dir + '/background.jpg';
```
Comment (siyangy) [inline modules/dreamview/frontend/src/renderer/ground.js]: I think the server part can be omitted?

File: modules/dreamview/frontend/src/renderer/ground.js (hunk 1 above)
Comment (unacao) [reply modules/dreamview/frontend/src/renderer/ground.js]: It is still called by render function of the main index

File: modules/dreamview/frontend/src/renderer/ground.js (hunk 2 above)
Comment (unacao) [reply modules/dreamview/frontend/src/renderer/ground.js]: Surely ok to remove. Was trying to leave a working code sample for dynamic import

File: modules/dreamview/frontend/src/renderer/ground.js (hunk 3 above)
Comment (unacao) [reply modules/dreamview/frontend/src/renderer/ground.js]: Server port needs to be fixed as the dist always serves from 8888 no matter dev or prod mode

**Pull request merged**